### PR TITLE
Collection of random checks

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Collection of random checks, focusing mainly on links, following recent books reorganisations.

I have also corrected some page titles or navigation bars, to align them (number of pages had completely different title to navigation bar links).

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/16614

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-intro.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [[contact-methods-intro]]
-= Contact Methods
+= Client Management Methods (Contact Methods)
```
Comment (jcayouette) [inline modules/client-configuration/pages/contact-methods-intro.adoc:2]: I woudld remove the (contact methods) bit. **Client Management Methods** seems like a good fit. Also keep in mind we should be swapping to sentence title syntax. 

Upcase only on the first item or proper names, etc.

**Client management methods**

File: modules/installation-and-upgrade/pages/client-requirements.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [[installation-client-requirements]]
-= Supported Client Systems
+= Supported Client Systems
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/client-requirements.adoc:2]:
```suggestion
= Supported Client Systems
```

File: modules/installation-and-upgrade/pages/install-proxy-unified.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [[installation-proxy-unified]]
-= {productname} {productnumber} Proxy
+= Install {productname} {productnumber} Proxy
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/install-proxy-unified.adoc:2]: Do we actually need prodnumber here?

File: modules/client-configuration/pages/delete-clients.adoc [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [[delete.clients]]
-= Delete Clients
+= Client Deletion
```
Comment (jcayouette) [inline modules/client-configuration/pages/delete-clients.adoc:2]: Perhaps **Remove clients**?

File: modules/installation-and-upgrade/pages/install-pubcloud.adoc [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [[install-pubcloud]]
-= Installing on a Public Cloud
+= Installing on Public Cloud
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/install-pubcloud.adoc:2]:
```suggestion
= Install on Public Cloud
```

Comment (jcayouette) [inline modules/installation-and-upgrade/pages/install-pubcloud.adoc:2]: Installation on the Public Cloud

File: modules/retail/pages/retail-saltboot-diagram.adoc [hunk 6]
```diff
@@ -1,5 +1,5 @@
 [[retail.saltboot.diagram]]
-= Saltboot Diagram
+= Terminal Boot process (Saltboot Diagram)
```
Comment (keichwa) [inline modules/retail/pages/retail-saltboot-diagram.adoc:2]:
```suggestion
= Terminal Boot Process (Saltboot Diagram)
```

**Review by jcayouette — CHANGES_REQUESTED** [review]
Looks good other than changes.

**Review by keichwa — APPROVED** [review]

**Pull request merged**